CONTRIBUTORS.md

```diff
@@ -5,4 +5,4 @@ Name | GitHub user | Description| Role
 Sacha Beniamine | @XachaB | | Other
 Erben Johansson | | publication author | Author
```

Comment:
We should probably include the full name here like we typically do, i.e. "Niklas Erben Johansson"

Reply:
thanks :)